cmd/gobmp/main.go

```diff
@@ -31,13 +31,13 @@ import (
 	"runtime"
 	"strconv"
 
+	"github.com/cisco-open/jalapeno/gobmp/arangodb"
```

Comment:
Why do you use fork of gobmp? I actively maintain gobmp as a opensource project, by forking you would need to duplicate efforts.